### ISSUE TITLE
Docs example err.(Wrong result)

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#570]
(Original issue 863 created by birdhackor on 2013-03-26T22:46:50.364138+00:00)

In http://pygments.org/docs/quickstart/

There has a example
```
#!python
>>> from pygments.lexers import guess_lexer, guess_lexer_for_filename
>>> guess_lexer('#!/usr/bin/python\nprint "Hello World!"')
<pygments.lexers.PythonLexer>
>>> guess_lexer_for_filename('test.py', 'print "Hello World!"')
<pygments.lexers.PythonLexer>

```

but i test 

```
#!python

>>> guess_lexer('#!/usr/bin/python\nprint "Hello World!"')
<pygments.lexers.NumPyLexer>
```

OS: Ubuntu 12.04 server

python: 2.7.3

pygments: 1.6

**Comment by Anteru** [comment]
(Original issue 863 last updated on 2013-05-20T05:41:27.201645+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2013-05-20T05:41:27.195507+00:00)

Closes #863: improve analyse_text of Python and Numpy lexers

→ <<cset d1aebb41a4de>>
